src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java.g

```diff
@@ -545,7 +545,7 @@ annotationExpression
 
 // Definition of a Java class
 classDefinition![AST modifiers]
-    :    c:"class" IDENT
+    :    c:"class" id:id
```

Comment:
Does this need to be named, or could it just be “id”?

Reply:
We need to assign the variable here to attach the IDENT AST to the CLASS_DEF AST.